api/src/main/java/uk/gov/hmcts/payment/api/controllers/CreditAccountPaymentController.java

```diff
@@ -77,7 +77,7 @@ public CreditAccountPaymentController(@Qualifier("loggingCreditAccountPaymentSer
             .siteId(creditAccountPaymentRequest.getSiteId())
             .build();
 
-        List<Fee> fees = creditAccountPaymentRequest.getFees().stream()
+        List<PaymentFee> fees = creditAccountPaymentRequest.getFees().stream()
```

Comment:
Are we renaming this because of name conflicts? If so, may be it's easy to fix fee-regsiter-model query with fully qualified name so that it'll fix for all other clients.